charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml

```diff
@@ -116,4 +114,8 @@ spec:
           hostPath:
             path: C:\k\secrets-store-csi-providers\
             type: DirectoryOrCreate
+{{- with .Values.windows.nodeSelector }}
+      nodeSelector:
```

Comment:
Shouldn't we require these nodeSelectors? ie. these should not be optional right?

Reply:
Updated the PR to keep the required `nodeSelector` in the template and also append the additional `nodeSelectors` that the user provides.